ehrapy/io/_read.py

```diff
@@ -488,8 +488,14 @@ def _get_non_existing_files(file: Path, download_dataset_name: str, backup_url:
     output_path_name = download_default_name.replace(archive_extension, "") if is_archived else download_default_name
     output_file_or_dir = ehrapy_settings.datasetdir / output_path_name
     moved_path = Path(str(output_file_or_dir)[: str(output_file_or_dir).rfind("/") + 1]) / download_dataset_name
-    shutil.move(output_file_or_dir, moved_path)  # type: ignore
-    file = moved_path
+    if moved_path.exists():
+        shutil.move(output_file_or_dir, moved_path)  # type: ignore
+        file = moved_path
+    elif not moved_path.exists():
```

Comment:
Can you add a comment above this to explain why we need this case? I know it but other people reading the code may not yet know the answer.